### PR TITLE
Update to Angular 7

### PR DESCRIPTION
**Pull request by pzalep1**
Updated to Angular 7

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -5,9 +5,9 @@ import { ClarkModule } from 'app/clark.module';
 import { environment } from '@env/environment';
 
 // Global rxjs imports
```
Comment (nrwinner) [inline src/main.ts]: Can we remove this comment if we're not importing anything under it?

File: tsconfig.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
   "compileOnSave": false,
   "compilerOptions": {
+    "importHelpers": true,
```
Comment (nrwinner) [inline tsconfig.json]: What is this?

File: src/main.ts (hunk 1 above)
Comment (pzalep1) [reply src/main.ts]: I missed this when I was looking through. I went back and removed it.

File: tsconfig.json (hunk 2 above)
Comment (pzalep1) [reply tsconfig.json]: image

**Pull request merged**